core/class-maxi-api.php

```
@@ -2908,8 +3256,21 @@ private function process_content_images($content)
                 // Download file to temp dir
-                $temp_file = download_url($url);
+                $temp_file = download_url($url, 15);
                 if (is_wp_error($temp_file)) {
+                    $this->import_warnings[] = [
+                        'code' => 'maxi_import_image_download_failed',
+                        'url' => esc_url_raw($url),
+                    ];
+                    return false;
+                }
+
+                if (filesize($temp_file) > 10 * MB_IN_BYTES) {
```

PHPStan (2.2.7)

[error] Constant MB_IN_BYTES not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)